cli.js

```diff
@@ -7,28 +7,44 @@ const npmName = require('npm-name');
 
 const cli = meow(`
 	Usage
-	  $ npm-name <name>
+	  $ npm-name <name> ...
 
 	Examples
 	  $ npm-name chalk
 	  ${logSymbols.error} ${chalk.bold('chalk')} is unavailable
 	  $ npm-name unicorn-cake
 	  ${logSymbols.success} ${chalk.bold('unicorn-cake')} is available
+	  $ npm-name chalk unicorn-cake
+	  ${logSymbols.error} ${chalk.bold('chalk')} is unavailable
+	  ${logSymbols.success} ${chalk.bold('unicorn-cake')} is available
 
-	Exits with code 0 when the name is available or 2 when taken
+	Exits with code 0 when all names are available or 2 when any names are taken
 `, {
 	string: ['_']
 });
 
-const input = cli.input[0];
+const input = cli.input;
 
-if (!input) {
+if (input.length === 0) {
 	console.error('Package name required');
 	process.exit(1);
 }
 
-npmName(input).then(available => {
-	const name = chalk.bold(input);
-	console.log(available ? `${logSymbols.success} ${name} is available` : `${logSymbols.error} ${name} is unavailable`);
-	process.exit(available ? 0 : 2);
+npmName.many(input).then(available => {
+	available.forEach(outputStatus);
+
+	let hasFalseValue = false;
+	for (const i of available.values()) {
+		if (!i) {
+			hasFalseValue = true;
+			break;
+		}
+	}
```

Comment:
You're right, forgot that `values()` returns an [iterator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Iterators_and_Generators). Your solution will do I guess. Can't think of a shorter version a.t.m.